ckanext/saml2auth/plugin.py

```diff
@@ -49,7 +50,8 @@ class Saml2AuthPlugin(plugins.SingletonPlugin):
     def get_helpers(self):
         return {
             'is_default_login_enabled':
-                h.is_default_login_enabled
+                h.is_default_login_enabled,
```

Comment:
There is no need here to use two lines for a single helper
Join them and let `'is_default_login_enabled': h.is_default_login_enabled,`